tests/post_training/data/wc_reference_data.yaml

```diff
@@ -22,8 +22,8 @@ tinyllama_int8_data_free_backend_TORCH:
   metric_value: 0.95624
   num_int4: 0
   num_int8: 312
-tinyllama_data_aware_gptq_backend_OV:
-  metric_value: 0.87134
+tinyllama_data_aware_gptq_scale_estimation_stateful_backend_OV:
+  metric_value: 0.8326
```

Comment:
hmm, it was much better. Wonder why similarity doesn't reflect improvement in the method. 
did you have a chance to look at the generated output? Does it contain empty strings or repetitions?

Reply:
The issue is related to OpenVINO. Updated the metric.